presto-iceberg/src/main/java/com/facebook/presto/iceberg/optimizer/IcebergPlanOptimizerProvider.java

```diff
@@ -25,19 +25,19 @@
 public class IcebergPlanOptimizerProvider
         implements ConnectorPlanOptimizerProvider
 {
-    private final ConnectorPlanOptimizer planOptimizer;
+    private final Set<ConnectorPlanOptimizer> planOptimizers;
```

Comment:
Same as above. Any reasons to update the type to a set? Looks like this change is unrelated to the parquet deference pushdown.

Reply:
As explained earlier, we need to pass the `IcebergParquetDereferencePushDown` and `IcebergPlanOptimizer` to the `IcebergPlanOptimizerProvider` class, and finally use it in the `IcebergPlanOptimizerProvider.getLogicalPlanOptimizers() `method.